create-rust-app/src/auth/controller.rs

```diff
@@ -642,11 +640,15 @@ pub fn forgot_password(
         )
         .unwrap();
 
-        let link = &format!("http://localhost:3000/reset?token={reset_token}");
-        mail::auth_recover_existent_account::send(mailer, &user.email, link);
+        let link = &format!("reset?token={reset_token}");
+        mailer
+            .templates
+            .send_recover_existent_account(mailer, &user.email, link);
```

Comment:
use template defined in Mailer.templates

base url removed because it's now controlled by the defaultmailtemplate struct